tutorials/p5js/_posts/2017-09-16-which-processing.md

```diff
@@ -121,7 +121,7 @@ For example, I use Processing to create these [movie color visualizations](/gall
 
 Even if your end goal is to get a programming job that's not directly tied to creative coding, I still think that creative coding is a great way to practice the fundamentals of coding, which you can then apply to whatever you want.
 
-To oversimplify my recommendation, I'd summarize my thoughts like this:
+To simplify my recommendation, I'd summarize my thoughts like this:
```

Comment:
Can we shorten this line to:

"I'd summarize my thoughts like this:"

(Get rid of the "simplify" part completely, since it's just extra fluff.)